packages/select/src/select.vue

```diff
@@ -866,7 +866,8 @@
           small: 32,
           mini: 28
         };
-        this.initialInputHeight = reference.$el.getBoundingClientRect().height || sizeMap[this.selectSize];
+        let input = [].filter.call(reference.$el.childNodes, item => item.tagName === 'INPUT')[0];
```

Comment:
```let input = reference.$el.querySelector('input')``` is better

Reply:
`const input = reference.$el.querySelector('input');` more better

Reply:
thanks, but i don't know how to modify this pr

Reply:
> thanks, but i don't know how to modify this pr

```
git commit --amend
git push origin dev -f
```

Reply:
thanks, but origin branch is deleted